warn/warn.go

```diff
@@ -640,6 +640,52 @@ func depsetIterationWarning(f *build.File, fix bool) []*Finding {
 	return findings
 }
 
+func argumentsOrderWarning(f *build.File, fix bool) []*Finding {
+	argumentType := func(expr build.Expr) int {
```

Comment:
this doesn't appear to need 'f'f or 'fix', so why an inline function rather than standalone?

Reply:
This file as well as `warn_bazel.go` has multiple independent warn/fix functions, IMO it looks better when the utility functions that are needed just for one of them are defined inside them, otherwise it's not clear which function belongs to which fix.